UDV-Core/src/Modules/ConsultDoc/DocumentBrowser.js

```diff
@@ -0,0 +1,273 @@
+
+import * as THREE from 'three';
```

Comment:
Same remark as for Demo.js concerning the lines longer than 80 columns.

Reply:
done

Reply:
Well, many lines still look much longer than 80 columns to me. Am I wrong ?

Reply:
Fixed